Dockerfile

```diff
@@ -1,5 +1,5 @@
 # 1) BUILD DEPENDENCIES
-FROM golang:1.15.6-alpine3.12 AS build-go
+FROM golang:1.15.8-alpine3.13 AS build-go
```

Comment:
There is a more recent golang docker image: **golang:1.16.0-alpine3.13**

Reply:
Good catch! Updated, tested and pushed